test/core/public/run.js

```diff
@@ -155,6 +155,21 @@ describe('axe.run', function () {
 				throw new Error('err');
 			});
 		});
+
+		it('is called after cleanup', function (done) {
```

Comment:
What is this test doing, exactly?

Reply:
It looks like it's monkey-patching `axe._runRules` and verifying the `cleanup` callback is invoked. Perhaps a more descriptive test name would help?